main.py

```diff
@@ -36,6 +37,17 @@ def main() -> int:
         print("Drone's destination is not final waypoint.")
 
         time.sleep(DELAY_TIME)
+    
+    IMAGE_LOG_PREFIX = "log_image"
+    device = CameraDevice(0, 100, IMAGE_LOG_PREFIX)
+
+    while True:
+        result, image = device.get_image()
+        if not result:
+            print("Failed to get image")
+            continue
+
+        time.sleep(DELAY_TIME)
```

Comment:
Use a new value, since this loop can run at a different period than the previous loop.